packages/mcp-memory/README.md

```
@@ -0,0 +1,169 @@
+pnpm --filter @corates/mcp-memory build
+
+# Run tests
+pnpm --filter @corates/mcp-memory test
+
+# Watch mode
+pnpm --filter @corates/mcp-memory dev
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)